src/main/java/net/openhft/chronicle/network/connection/TcpChannelHub.java

```diff
@@ -63,11 +64,24 @@
  * ), when the server responds to the client, its expected that the server sends back the tid as the very first field in the message. The
  * TcpChannelHub will look at each message and read the tid, and then marshall the message onto your appropriate client thread. Created by Rob Austin
  */
+@Deprecated(/* for removal in x.26 */)
```

Comment:
I vote we remove in x.25

Reply:
Suggest we create an issue so customers have visibility on this deprecation (and future removal)

Reply:
Yep agree on both, will create the issue and update the commit message once it's been green lighted by the team